eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -95,10 +96,11 @@ object Graph {
                         pathsToFind: Int,
                         wr: Option[WeightRatios],
                         currentBlockHeight: Long,
-                        boundaries: RichWeight => Boolean): Seq[WeightedPath] = {
+                        boundaries: RichWeight => Boolean,
+                        isRelay: Boolean): Seq[WeightedPath] = {
```

Comment:
I would rename it to something that makes what it does more clear, for example `includeLocalChannel(Cost|Weight)` or `countLocalChannel(Cost|Weight)` (or on the contrary, if you find it more clear, that parameter could be for *ignoring* local channel costs: `ignoreLocalChannel(Cost|Weight)`)

Reply:
Renamed to includeLocalChannelCost, I agree it's a better name.